ebmlite/schemata/mide_manifest.xml

```diff
@@ -154,6 +154,8 @@
         <!-- Battery charger ICs -->
         <MasterElement name="ChargerMCP73837" id="0x4D60" mandatory="0" multiple="0" minver="1">Indicates the presence of a MCP73837 battery charger.</MasterElement>
         <MasterElement name="ChargerMAX14747" id="0x4D61" mandatory="0" multiple="0" minver="1">Indicates the presence of a MAX14747 battery charger.</MasterElement>
+        <MasterElement name="NoCharger" id="0x4D61" mandatory="0" multiple="0" minver="1">Indicates the presence of no battery charger or just a single cell battery.</MasterElement>
+
```

Comment:
Big issue, we can't have 2 elements with the same ID, so can you change the NoCharger to maybe 0x4D62? Please make sure that ID is not used elsewhere
Smaller issue, can you change the name to match the style of similar elements. Maybe change it to "ChargerNone"